Random Users/View Controllers/UserDetailViewController.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  UserDetailViewController.swift
+//  Random Users
+//
+//  Created by conner on 5/15/20.
+//  Copyright © 2020 Erica Sadun. All rights reserved.
+//
+
+import UIKit
+
+class UserDetailViewController: UIViewController {
+    @IBOutlet weak var image: UIImageView!
+    @IBOutlet weak var username: UILabel!
+    @IBOutlet weak var phone: UILabel!
+    @IBOutlet weak var email: UILabel!
```

Comment:
```suggestion
    @IBOutlet var image: UIImageView!
    @IBOutlet var username: UILabel!
    @IBOutlet var phone: UILabel!
    @IBOutlet var email: UILabel!
```

Apple changed their mind again...